.github/workflows/build-swift.yml

```diff
@@ -0,0 +1,102 @@
+---
+name: Build Swift Package
+
+on:
+  push:
```

Comment:
@Hinton Can we trigger only on changes in the `languages/swift` repository? Or should we trigger on every change in the repo?

Reply:
It should be every change. I don't expect changes to languages/swift to be common.